devscripts/generate_fake_sessions.py

```
@@ -0,0 +1,122 @@
+NUM_ROWS = 10000  # Set the number of rows to generate
+
+import os
+import random
+import uuid
+from datetime import timedelta
+from django.utils import timezone
+
+from kolibri.utils.cli import initialize
+initialize()
+import sqlite3
+import os
```

Ruff (0.8.2)

Redefinition of unused `os` from line 19

Remove definition: `os`

(F811)